### PR TITLE
Add Croatian translation

### PR DESCRIPTION
**Pull request by milotype**

### CODE REVIEW

**Comment by ferraridamiano** [comment]
Hi @milotype and thank you very much for all this work. As you might know I'm working on [version 3.0](https://github.com/ferraridamiano/ConverterNOW/milestone/3) of this app and this means that there will be some other strings. I just commit them on this PR. Can you translate them? I left them in english. Once done I will merge this PR. Thank you!

File: lib/l10n/app_hr.arb [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  "light": "Svijetla",
+  "system": "Sustav",
+  "amoledDarkTheme": "AMOLED tamna tema",
+  "copy": "Kopiraj",
+  "more": "Više",
+  "language": "Jezik",
+  "reorderProperties": "Reorder properties",
+  "reorderUnits": "Reorder units",
+  "chooseProperty": "Choose a property",
+  "reorderProperty": "Reorder {property}",
```
Comment (ferraridamiano) [inline lib/l10n/app_hr.arb:41]: I'm referring to these one

Comment (milotype) [reply lib/l10n/app_hr.arb:41]: OK, I’ll translate those strings. What about line 46 `"example": "Lenght"`? Should “Length” be translated?

Comment (ferraridamiano) [reply lib/l10n/app_hr.arb:41]: Thank you! Nope, it is not necessary. It is just an example for the `property` placeholder. I wrote it just for context. "Property" is a generic word, but with it I mean property such that lenght, area, currencies, time, etc

Comment (milotype) [reply lib/l10n/app_hr.arb]: OK, done!

**Comment by ferraridamiano** [comment]
Awesome! Merged! Thank you @milotype !

**Pull request merged**